autoscale/src/main/java/com/sequenceiq/periscope/controller/DistroXAutoScaleClusterV1Controller.java

```diff
@@ -119,10 +129,18 @@ private DistroXAutoscaleClusterResponse updateClusterAutoScaleConfig(Long cluste
                 alertController.createTimeAlerts(clusterId, autoscaleClusterRequest.getTimeAlertRequests());
                 asClusterCommonService.setAutoscaleState(clusterId, autoscaleClusterRequest.getEnableAutoscaling());
             });
+            createHistoryAndNotifyConfigChange(clusterId);
         } catch (TransactionService.TransactionExecutionException e) {
             throw e.getCause();
         }
 
         return createClusterJsonResponse(clusterService.findById(clusterId));
     }
+
+    private void createHistoryAndNotifyConfigChange(long clusterId) {
+        Cluster cluster = clusterService.findById(clusterId);
+        History history = historyService.createEntry(ScalingStatus.CONFIG_UPDATED,
+                "Autoscaling config has been updated for the cluster.", cluster);
+        notificationSender.send(cluster, history);
```

Comment:
Updated to include HostGroup and ScalingType in the historyEvent for CONFIG_UPDATED.  This is a valid history event along with Enabling\Disabling Autoscaling and would be required in case of support issues.
 
There is little noise because UI updates on each change basically for Load-Based Autoscaling, specific event CONFIG_UPDATED can be filtered out while showing in UI if it becomes too noisy.